gnucash/gnucash-cli.cpp

```diff
@@ -150,6 +152,10 @@ Gnucash::GnucashCli::start ([[maybe_unused]] int argc, [[maybe_unused]] char **a
                 return Gnucash::run_report(m_file_to_load, m_report_name,
                                            m_export_type, m_output_file);
         }
+        else if (*m_report_cmd == "list")
+                return Gnucash::report_list ();
```

Comment:
I initially was confused for not passing a file name to this command, but eventually it dawned on my saved reports are not stored in our data files yet so we don't need it. That also explains your comment in the PR comment. Perhaps add a comment in the code as well pointing this out. It's not *that* obvious from a distance.